### PR TITLE
Upgrade camshaft to 0.55.0

### PR DESCRIPTION
**Pull request by dgaubert**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,8 +1,11 @@
 # Changelog
 
-## 3.7.2
+## 3.8.0
```
Comment (ethervoid) [inline NEWS.md]: Add the real date not the template one

Comment (dgaubert) [inline NEWS.md]: it should be done at release time. shouldn't it?

Comment (dgaubert) [inline NEWS.md]: I mean, we want release today, but shit happens..

Comment (ethervoid) [inline NEWS.md]: Ok, I thought this was the branch for release. Sorry!

**Pull request merged**